dkg/dkg.go

```diff
@@ -67,6 +66,11 @@ func Run(ctx context.Context, conf Config) (err error) {
 		return err
 	}
 
+	network, err := forkVersionToNetwork(def.ForkVersion)
```

Comment:
I think we should do it the other way round, provide a network and map it to fork version. Requiring people to input fork versions is not a great UX.

Reply:
agreed

Reply:
Done!
Modified in createdkg command flags